repos/system_upgrade/el7toel8/libraries/fetch.py

```diff
@@ -0,0 +1,74 @@
+import os
+
+import requests
+
+from leapp.exceptions import StopActorExecutionError
+from leapp.libraries.common.config import get_env
+from leapp.libraries.stdlib import api
+
+SERVICE_HOST_DEFAULT = "https://cert.cloud.redhat.com"
+PROXY_HOST_DEFAULT = "http://squid.corp.redhat.com:3128"
+REQUEST_TIMEOUT = 5
+
+
+def _raise_error(local_path, details):
+    """
+    If the file acquisition fails in any way, throw an informative error to stop the actor.
+    """
+    summary = ("Data file {lp} is invalid or could not be retrieved.\n"
+               "Error details: {d}\n\n"
+               "Read documentation at: https://access.redhat.com/articles/3664871"
+               " for more information about how to retrieve the file."
+               ).format(lp=local_path, d=details)
+
+    raise StopActorExecutionError(summary)
```

Comment:
this is wrong. Details should not be part of the title.. see e.g. the AddUpgradeBootEntry actor how the title and details are provided in error messages.

Reply:
Should the documentation link also go into details?

Reply:
Moved some stuff into details.